packages/browser-agent/src/index.ts

```diff
@@ -44,6 +44,7 @@ export class BrowserAgent {
 
   }
   public noticeError = (err: Error | String, customAttributes?: Object) => this._api.noticeError(err, customAttributes);
+  public addPageAction = (name: String, customAttributes?: Object) => this._api.addPageAction(name, customAttributes);
```

Comment:
This likely needs some architectural changes, but the reason it is layered like this, is because there is a chance that the customer does not import a certain feature, and the API and the feature set are not aligned.